evalml/tests/component_tests/test_components.py

```diff
@@ -1884,3 +1885,86 @@ def test_handle_nullable_types(
             y_d.ww.logical_type,
             tuple(y_incompatible_ltypes),
         )
+
+
+@pytest.mark.parametrize(
+    "component_class",
+    [
+        comp
+        for comp in all_components()
+        if not (
+            comp._integer_nullable_incompatibilities
+            or comp._boolean_nullable_incompatibilities
+        )
+    ],
+)
+@pytest.mark.parametrize(
+    "nullable_y_ltype",
+    ["BooleanNullable", "IntegerNullable", "AgeNullable"],
+)
+# --> this is 180 tests - is it overkill?
+def test_components_support_nullable_types(
```

Comment:
I feel like we want the tests for making sure our compatible components are actually compatible should be exhaustive. This does add ~180 tests, though, so I just want to make sure there's visibility around that.